ipatests/conftest.py

```diff
@@ -106,8 +106,9 @@ def pytest_cmdline_main(config):
 
     # XXX workaround until https://fedorahosted.org/freeipa/ticket/6408 has
     # been resolved.
-    if ipaserver is not None and installutils.is_ipa_configured():
-        api.finalize()
+#    if ipaserver is not None and installutils.is_ipa_configured():
```

Comment:
This change seems unrelated and should not be required. In fact it is going to break Python package tests and make fastcheck.